oci/layout/assets/certs/accepted/cacert.crt

```diff
@@ -0,0 +1,13 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Nit: Please use `fixtures` for the name of the subdirectory containing test fixtures, to be consistent with the other subpackages.

Reply:
:+1:
going to update it